bot.py

```diff
@@ -345,6 +366,11 @@ async def handle_message(self, message):
         # Execute the command.
         await handler(message, command_content)
 
+    def get_spotify_client(self):
+        """Get Spotify client"""
+        creds = spotipy.oauth2.SpotifyClientCredentials()
```

Comment:
When running the bot, I get an error which traces back to this line:
```
Traceback (most recent call last):
  File "/home/michael/files/projects/the-lone-dancer/venv/lib/python3.10/site-packages/discord/client.py", line 343, in _run_event
    await coro(*args, **kwargs)
  File "/home/michael/files/projects/the-lone-dancer/bot.py", line 50, in on_message
    self.clients[message.guild] = MusicBot(message.guild, self.loop, self.user)
  File "/home/michael/files/projects/the-lone-dancer/bot.py", line 142, in __init__
    self.spotify = self.get_spotify_client()
  File "/home/michael/files/projects/the-lone-dancer/bot.py", line 371, in get_spotify_client
    creds = spotipy.oauth2.SpotifyClientCredentials()
  File "/home/michael/files/projects/the-lone-dancer/venv/lib/python3.10/site-packages/spotipy/oauth2.py", line 201, in __init__
    self.client_id = client_id
  File "/home/michael/files/projects/the-lone-dancer/venv/lib/python3.10/site-packages/spotipy/oauth2.py", line 94, in client_id
    self._client_id = _ensure_value(val, "client_id")
  File "/home/michael/files/projects/the-lone-dancer/venv/lib/python3.10/site-packages/spotipy/oauth2.py", line 70, in _ensure_value
    raise SpotifyOauthError(msg)
spotipy.oauth2.SpotifyOauthError: No client_id. Pass it or set a SPOTIPY_CLIENT_ID environment variable.
```

Maybe you forgot to add the environment variable to `.env.example`?

Reply:
Yes. Added the environment variables to `.env.example`. Also updated README.

Reply:
I can't test this right now, but won't the bot still crash if we don't supply a valid spotify token?

Reply:
I modified the `get_spotify_client` code to check for this error, and log it, but return `None` instead. This means everything will work, except fetching spotify stuff. Trying to play spotify stuff will return a :robot: message to the user, and log it as an error every time in the logs.